### PR TITLE
CITY-12: New redesign with assets from Avtar

### PR DESCRIPTION
**Pull request by grrrero**
Redesign featuring Avtar's assets, re-arranging and breaking up the blocks of text on the homepage, adding different colour palette, and redesign of the Ideas in Progress sub-pages.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -9,49 +9,59 @@ body, html {
...
-	background-size: cover;
-	font-family: 'Fira Sans', sans-serif;
-	font-weight: 300;
+	background-color: #fff;
+	font-family: 'Work Sans', sans-serif;
+	font-weight: 400;
+	color: #000;
 }
 
 a {
```
Comment (colinbdclark) [inline css/style.css]: It looks like you've got inconsistent indenting throughout this file.

File: index.html [hunk 2]
```diff
@@ -46,57 +46,61 @@
...
+				</div>
+				<div class="contentBlock">
+					<p>Inclusive design assumes no predetermined end point and no generalized success criteria, instead inviting diverse participants to determine the outcomes and measures collectively. The inclusive design framework is intended to be more like a trellis that supports organic growth and provides a foundation from which to innovate and evolve. The three dimensions of the framework are:</p>
+
+					<ol>
+						<li>Recognize, respect, and design for human uniqueness and variability.</li>
+						<li>Use inclusive, open & transparent processes, and co-design with people who have a diversity of perspectives, including people that can’t use or have difficulty using the current environments.</li>
```
Comment (colinbdclark) [inline index.html]: Another missing entity reference here.

File: ideas/for-an-inclusive-quayside/index.html [hunk 3]
```diff
@@ -42,53 +42,59 @@
...
-            <a href="http://idrc.ocadu.ca/" class="logo-idrc" title="Inclusive Design Research Centre">Inclusive Design Research Centre</a>
-        </div>
-        <p>This website is maintained and created by the <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a>. It is distributed under the <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode">Creative Commons Attribution, Non-Commercial, Share-Alike 4.0 International license</a>.</p>
+      <div id="footer">
+          <div class="logoContainer">
+              <a href="http://idrc.ocadu.ca/" class="logo-idrc" title="Inclusive Design Research Centre">Inclusive Design Research Centre</a>
+          </div>
+          <p>This website is maintained and created by the <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a>.<br />
+          It is distributed under the <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode">Creative Commons Attribution, Non-Commercial, Share-Alike 4.0 International license</a>.<br />
+        Background image of city of Toronto credited to <a href="https://chuckmantorontonostalgia.wordpress.com/">John Chuckman</a>.</p>
```
[the remark this answers is not in the document]
Comment (grrrero) [reply ideas/for-an-inclusive-quayside/index.html]: How can I word it better?

[the remark this answers is not in the document]
Comment (colinbdclark) [reply ideas/for-an-inclusive-quayside/index.html]: What I mean is, there aren't any images on this page whatsoever. :)

File: index.html [hunk 4]
```diff
@@ -110,8 +114,10 @@ <h3><a href="ideas/co-design-draft-proposal">IDRC Inclusive Co-Design Draft Prop
 	<div id="footer">
 		<div class="logoContainer">
 		 	<a href="http://idrc.ocadu.ca/" class="logo-idrc" title="Inclusive Design Research Centre">Inclusive Design Research Centre</a>
-		</div>
-		<p>This website is maintained and created by the <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a>. It is distributed under the <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode">Creative Commons Attribution, Non-Commercial, Share-Alike 4.0 International license</a>.</p>
+		 </div>
+		<p>This website is maintained and created by the <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a>.<br />
+			It is distributed under the <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode">Creative Commons Attribution, Non-Commercial, Share-Alike 4.0 International license</a>.<br />
+			Background image of city of Toronto credited to <a href="https://chuckmantorontonostalgia.wordpress.com/">John Chuckman</a>.</p>
```
Comment (colinbdclark) [inline index.html]: Same here.

File: ideas/for-an-inclusive-quayside/index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (grrrero) [reply ideas/for-an-inclusive-quayside/index.html]: Oh -- right not on those pages. Will remove.

File: index.html [hunk 5]
```diff
@@ -131,4 +137,4 @@ <h3><a href="ideas/co-design-draft-proposal">IDRC Inclusive Co-Design Draft Prop
 	</script>
 
 </body>
-</html>
+</html>
```
Comment (colinbdclark) [inline index.html]: Generally, you should configure your editor to leave a blank line at the end of files automatically for you.

Comment (colinbdclark) [reply index.html]: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Comment (grrrero) [reply index.html]: https://github.com/mattst/sublime-single-trailing-new-line

?

Comment (colinbdclark) [reply index.html]: I'm not a Sublime user myself, but that's probably a fine plugin if the default settings don't work for you. Perhaps try them for a while first and see how it goes?

Comment (grrrero) [reply index.html]: Yep just installed it and will see!

**Pull request merged**